.github/workflows/semantic_release.yml

```
@@ -0,0 +1,77 @@
+# This workflow runs semantic release, bumps, generates changelog, and tags the project
+name: Semantic Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: "Are you releasing a new version? (y/n)"
```

Let's do either

```suggestion
      release-new-version:
        description: "Are you releasing a new version? (y/n)"
```

or

```suggestion
      dry-run:
        description: "Is this a dry run to validate semantic-release behaves as expected? (y/n)"
```

Right now, the Y/N are in opposition